easybuild/easyblocks/o/openblas.py

```diff
@@ -26,7 +26,14 @@ def configure_step(self):
             'USE_OPENMP': '1',
             'USE_THREAD': '1',
         }
-        if LooseVersion(self.version) < LooseVersion('0.3.6') and get_cpu_architecture() == POWER:
+
+        if 'TARGET' in self.cfg['buildopts']:
+            # Add any TARGET in buildopts to default_opts, so it is passed to testopts and installopts
+            for buildopt in self.cfg['buildopts'].split():
+                optpair = buildopt.split('=')
+                if optpair[0] == 'TARGET':
```

Comment:
Usually when we use a "magic string" in more than one place, we introduce a constant.

```python
TARGET = 'TARGET'
```

```python
if (TARGET + '=') in self.cfg['buildopts']:
    ...
    if optpair[0] == TARGET:
```

May look like overkill, but it helps avoid bugs.

Reply:
Makes perfect sense, fixed :+1: